.goreleaser.yaml

```diff
@@ -42,3 +42,5 @@ brews:
     commit_author:
       name: rachfop
       email: prachford@icloud.com
+    custom_block: |
+      deprecate! date: "2024-05-08", because: "has been replaced by humctl score deploy. For more information visit https://developer.humanitec.com/platform-orchestrator/reference/cli-references/#score-integration."
```

Comment:
`deprecate!` as described in https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae

Reply:
that's very useful!